src/frontend/komponenter/Task/KommenteringModal/kommenteringModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { Knapp } from 'nav-frontend-knapper';
+import Modal from 'nav-frontend-modal';
+import { Select, Textarea } from 'nav-frontend-skjema';
+import 'nav-frontend-tabell-style';
+import { Element, Undertittel } from 'nav-frontend-typografi';
+import React, { FC, useState } from 'react';
+import { avvikstyper, ITask } from '../../../typer/task';
+import { useTaskContext } from '../../TaskProvider';
+import { Alert, Checkbox } from '@navikt/ds-react';
+
+interface IProps {
+    settÅpen: (åpen: boolean) => void;
+    task: ITask;
+    åpen: boolean;
+}
+
+const KommenteringModal: FC<IProps> = ({ settÅpen, task, åpen }) => {
+    const { kommenter } = useTaskContext();
+    const [tilManuellOppfølging, settTilManuellOppfølging] = useState<boolean>(false);
+    const [kommentar, settKommentar] = useState('');
+    const [feilMelding, settFeilMelding] = useState('');
+
+    return (
+        <Modal
+            contentClass={'kommentering'}
+            isOpen={åpen}
+            closeButton={true}
+            onRequestClose={() => {
+                settÅpen(!åpen);
+            }}
+            contentLabel="Kommenter"
+        >
+            <Undertittel children={`Kommenter`} />
+
+            <br />
+            <Element children={'Legg til kommentar og velge hvis task skal bli manuelloppfølgt'} />
+
+            <form
+                onSubmit={(event) => {
+                    kommenter(
+                        {
+                            taskId: task.id,
+                            settTilManuellOppfølging: tilManuellOppfølging,
+                            kommentar: kommentar,
+                        },
+                        () => {
+                            settÅpen(false);
+                            settFeilMelding('');
+                        },
+                        (error: string) => {
+                            settFeilMelding(error);
+                        }
```

Comment:
Trekk gjerne ut som navnede funksjoner så blir det lettere å lese koden.

Tenker at det meste av javascript inni JSX-en kan trekkes ut til én funksjon hvor logikken ligger. 
Da blir det lettere å lese flyten i JSX-en.

Reply:
I don't know if I understood the comment. Please see my fix. Sorry for my Nowegian:)